swo_aws_extension/management/commands/upload_journal.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from mpt_extension_sdk.core.utils import setup_client
+from requests.exceptions import HTTPError
+
+from swo_mpt_api import MPTAPIClient
+from swo_mpt_api.models.hints import Journal
+
+
+class Command(BaseCommand):
+    """
+    Upload a journal file to the marketplace
+
+    Upload a file by providing authorization id or journal id.
+
+    Arguments:
```

Comment:
```suggestion
   Args:
```
We should use Args: instead of Arguments to follow our code style conventions. You can configure by default in the IDE